### DISCUSSION TITLE
What are the acceptable image formats for front and rear elevation images?

### DISCUSSION THREAD

**Discussion by colinmacgiolla** [netbox-community/devicetype-library#1739] [Q&A] [answered]
Looking at the README, we have 

> The file name must also adhere to <VALUE_IN_SLUG>.front.<acceptable_format>

 for the front/rear elevation images, but what are the acceptable formats?

Everything so far seems to be .png, but would JPEGs (.jpg) also be considered "acceptable"?

**Comment by danner26** [comment, marked as answer]
I would suggest png if possible, but you should be able to use almost any type. The following are what comes up as options when uploading an image to NetBox: `pjp, jfif, svgz, jpeg, ico, tiff, avif, svg, jpg, apng, tif, gif, png ,xbm, pjpeg, bmp and webp`
